dataland-keycloak/dataland_theme/login/templates/login-update-password.ftl

```diff
@@ -0,0 +1,83 @@
+<#import "dataland_template.ftl" as layout>
+<@layout.registrationLayout displayMessage=false; section>
+    <#if section = "header">
+        Reset password
+    <#elseif section = "backUrl">/
+    <#elseif section = "backName">HOME
+    <#elseif section = "form">
+
+        <div>
+            <span>Enter a new password for your Dataland account.</span>
+        </div>
+
+        <form id="kc-passwd-update-form" class="${properties.kcFormClass!}" action="${url.loginAction}" method="post">
+
+            <input type="text" id="username" name="username" value="${username}" autocomplete="username"
+                   readonly="readonly" style="display:none;"/>
+            <input type="password" id="password" name="password" autocomplete="current-password" style="display:none;"/>
+
+
+            <div class="${properties.kcFormGroupClass!} pt-4">
+                <div class="${properties.kcInputWrapperClass!} p-relative-position">
+                    <input type="password" id="password-new" name="password-new"
+                           required autofocus autocomplete="new-password"
+                           aria-invalid="<#if messagesPerField.existsError('password','password-confirm')>true</#if>"
+                    />
+                    <label for="username">New password</label>
+
+                    <#if messagesPerField.existsError('password')>
```

Comment:
I really can't tell you, since this is the unchanged and original default Keycloak code here.
We didn't change or customize it.
So if this code was wrong, the keycloak developers probably would have already noticed it.